lib/Tests/BluetoothEngineTests/WatchAdvertisementsTests.swift

```diff
@@ -0,0 +1,63 @@
+import Bluetooth
+@testable import BluetoothAction
+import BluetoothClient
+import BluetoothMessage
+import Foundation
+import JsMessage
+import Testing
+
+extension Tag {
+    @Tag static var watchAdvertisements: Self
+}
+
+@Suite(.tags(.watchAdvertisements))
+struct WatchAdvertisementsRequestTests {
+
+    @Test
+    func decode_withValidInputs_succeeds() {
+        let uuid = UUID(n: 0)
+        let body: [String: JsType] = [
+            "enable": .number(true),
+            "uuid": .string(uuid.uuidString),
+        ]
+        let request = WatchAdvertisementsRequest.decode(from: body)
```

Comment:
total nit, but I think it makes the tests slightly faster to parse. You typealias WatchAdertisementsRequest to "sut"